src/v3/src/components/Link/Link.tsx

```diff
@@ -43,21 +43,28 @@ const Link: UISchemaElementComponent<{
   };
 
   return (
-    <Box marginTop={4}>
-      {
-        typeof href === 'undefined' ? (
-          <LinkMui
-            // eslint-disable-next-line no-script-url
-            href="javascript:void(0)"
-            onClick={onClick}
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {...(dataSe && { 'data-se': dataSe } )}
-          >
-            {getLabelName(label)}
-          </LinkMui>
-        ) : <LinkMui href={href}>{getLabelName(label)}</LinkMui>
-      }
-    </Box>
+    typeof href === 'undefined' ? (
+      <LinkMui
+        // eslint-disable-next-line no-script-url
+        href="javascript:void(0)"
+        onClick={onClick}
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...(dataSe && { 'data-se': dataSe } )}
+        sx={{
+          verticalAlign: 'middle',
+        }}
```

Comment:
Is this something all of the `Link` elements should have in the odyssey theme? We may want to talk to the Odyssey team about this and see if there's something generally they can add to the theme.

Reply:
It might be necessary since it doesn't line up with inline text

Reply:
alright, let's go with this 🚀